pkg/github/github.go

```diff
@@ -31,12 +29,9 @@ const AppStudioAppDataOrg = "redhat-appstudio-appdata"
 
 // GenerateNewRepositoryName creates a new gitops repository name, based on the following format:
 // <display-name>-<partial-hash-of-clustername-and-namespace>-<random-word>-<random-word>
-func GenerateNewRepositoryName(displayName, namespace, clusterName string) string {
+func GenerateNewRepositoryName(displayName, userUniqueHash string) string {
```

Comment:
Which display name is used here?

Reply:
Not sure. This refactor does not involve argument `displayName`, so I didn't pay attention to it.